migration_sql.go

```diff
@@ -95,7 +96,8 @@ func getSQLStatements(r io.Reader, direction bool) (stmts []string, tx bool) {
 		}
 
 		if _, err := buf.WriteString(line + "\n"); err != nil {
-			log.Fatalf("io err: %v", err)
+			e = fmt.Errorf("io err: %v", err)
+			return
```

Comment:
Personally, I'd prefer `return nil, false, err` over the named return arguments.

Reply:
great, thanks

We gotta improve the old goose code along the way, these parts are years and years old :)

Reply:
On that point, are you happy to go with unnamed return types (in this PR anyway)?

Generally not a fan of named return values as it can obfuscate where the variable was declared when it's much easier to see if the variable is initialised in the method and then explicitly returned instead of the implicit return that can go with named return values.

Reply:
yes, I'd love to see the unnamed return types :)

Reply:
btw: I'd really appreciate if you had time to go through the whole codebase to make these changes (bubbling up errors, unnamed return types) in the following PRs, if you were interested